doc/source/theory/dem/dem.rst

```diff
@@ -82,26 +82,26 @@ The relative velocities calculated to allow updating the tangential overlap are
      - Linear model definitions
      - Nonlinear viscoelastic model definitions
    * - Normal spring constant
-     - :math:`k_n = \frac{16}{15}\sqrt{R_{e}}Y_{e}\left(\frac{15m_{e}V^2}{16\sqrt{R_{e}}Y_{e}}\right)^{0.2}`
+     - :math:`k_n = 1.2024(m^{0.5}E)Y_{e}^{2}R_{e}\nu_{e})^{0.4}`
      - :math:`k_n = \frac{4}{3}Y_{e}\sqrt{R_{e}\delta_n}`
    * - Tangential spring constant
      - :math:`\eta_n = \sqrt{\frac{4m_{e}k_n}{1+\left(\frac{\pi}{\ln{e}}\right)^2}}`
      - :math:`\eta_n = -2\sqrt{\frac{5}{6}}\beta\sqrt{S_nm_{e}}`
    * - Normal damping model constant
-     - :math:`k_t = k_n`
+     - :math:`k_t = 0.4 k_n`
      - :math:`k_t = 8G_{e}\sqrt{R_{e}\delta_n}`
    * - Tangential damping model constant
-     - :math:`\eta_t = \eta_n`
+     - :math:`\eta_t = \sqrt{\frac{4m_{e}k_t}{1+\left(\frac{\pi}{\ln{e}}\right)^2}}`
      - :math:`\eta_t = -2\sqrt{\frac{5}{6}}\beta\sqrt{S_tm_{e}}`
```

Comment:
Can you flip the Tangential spring constant and Normal damping model constant parameters in the table? Those are wrong with the models in columns. Shame on that person that made the mistake.